### PR TITLE
Implement Login - TC01, TC02, TC03, TC04 and Register - TC00

### PR DESCRIPTION
**Pull request by sattdn-23-01-03-NguyenNhatNam**
Please review me.

### CODE REVIEW

File: src/main/java/page/LoginPage.java [hunk 1]
```diff
@@ -5,41 +5,56 @@
...
     private By btnLogin = By.xpath("//input[@type='submit']");
     private By msgWelcome = By.xpath("//div[@class='account' and contains(normalize-space(),'Welcome ')]");
-
     private By msgErrorLoginForm = By.xpath("//p[@class='message error LoginForm']");
     private By msgErrorValidField = By.xpath("//label[@class='validation-error' and @for ='username']");
 
+    public WebElement getEmailField() {
+        return Constant.WEBDRIVER.findElement(txtEmail);
+    }
+    public void enterEmail(String email){
```
Comment (cheanhminh98) [inline src/main/java/page/LoginPage.java]: please format code, use "space" between () and {}

File: src/main/java/page/LoginPage.java [hunk 2]
```diff
@@ -5,41 +5,56 @@
...
+        getEmailField().sendKeys(email);
+    }
 
-    public void enterEmail(String email) {
-        WebElement emailTxtBox = Constant.WEBDRIVER.findElement(txtEmail);
-        emailTxtBox.sendKeys(email);
+    public WebElement getPasswordField(){
```
Comment (cheanhminh98) [inline src/main/java/page/LoginPage.java]: same as above

File: src/main/java/page/RegisterPage.java [hunk 3]
```diff
@@ -15,37 +15,49 @@ public class RegisterPage {
...
     }
-
 
     public void clickRegister() {
-        WebElement registerClick = Constant.WEBDRIVER.findElement(btnRegister);
-        registerClick.click();
+        getRegisterBtn().click();
     }
+
     public void srollPage() throws InterruptedException {
```
Comment (cheanhminh98) [inline src/main/java/page/RegisterPage.java]: 1. please edit the name to scrollPage not srollPage
2. we should create it in the another class (example: ultilities,..)

File: src/test/java/railway/RegisterTest.java [hunk 4]
```diff
@@ -12,17 +12,17 @@ public class RegisterTest extends BaseTest {
     public HomePage homePage = new HomePage();
 
     @Test
-    public void TC() throws InterruptedException {
+    public void TC00() throws InterruptedException {
```
Comment (cheanhminh98) [inline src/test/java/railway/RegisterTest.java:15]: 1. What is this mean "TC00"?
2. Should not use "InterruptedException" in a test case

File: src/main/java/page/RegisterPage.java (hunk 3 above)
Comment (sattdn-23-01-03-NguyenNhatNam) [reply src/main/java/page/RegisterPage.java]: I added in Utilities class

File: src/test/java/railway/RegisterTest.java (hunk 4 above)
Comment (sattdn-23-01-03-NguyenNhatNam) [reply src/test/java/railway/RegisterTest.java:15]: Last week, the homework create a test case "User registers successfully then login with this account". It's not have in Set1 Testcase. So I set it "TC00"

**Comment by sattdn-23-01-03-NguyenNhatNam** [comment]
@cheanhminh98 
I changed my branch name to impl_Login_TC01_02_03_04ANDRegister_TC00. So I closed this PR!

**Pull request closed without merging**